client/src/actions/index.ts

```diff
@@ -36,29 +40,27 @@ async function userColorsFetchAndLoad(dispatch: any) {
   );
 }
 
-async function schemaFetch() {
-  return fetchJson("schema");
+async function schemaFetch(): Promise<{ schema: Schema }> {
+  return fetchJson<{ schema: Schema }>("schema");
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-async function configFetch(dispatch: any) {
-  return fetchJson("config").then((response) => {
-    const config = { ...globals.configDefaults, ...response.config };
+async function configFetch(dispatch: Dispatch) {
```

Comment:
why not type the return as `Promise<Config>`?

Reply:
Fixed!